package.json

```diff
@@ -80,11 +79,11 @@
     "react": "^15.4.2",
     "react-addons-test-utils": "15.4.2",
     "react-helmet": "^3.3.2",
-    "react-intl": "^2.2.3",
-    "react-redux": "^5.0.2",
-    "react-router": "^3.0.2",
-    "redux": "^3.5.2",
-    "rxjs": "^5.0.3",
+    "react-intl": "2.2.3",
+    "react-redux": "5.0.2",
+    "react-router-dom": "4.0.0-beta.5",
```

Comment:
Yeah I don't love the fact that we need the same package listed twice in order to correctly run tests, but in a 'pure' sense, those peer dependencies are necessary for the tests to run in dev, so they technically should be in both.

I'd be interested in figuring out a better way, but I'm hesitant to add another dependency to deal with dependency challenges... feel free to open a ticket